src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -326,7 +326,7 @@ function TextArea({
       <textarea
         className={classnames(
           'border rounded p-2',
-          'text-color-text-light bg-grey-0',
+          'bg-grey-0 text-grey-8 placeholder:text-grey-6',
```

Comment:
This uses a darker grey both for the actual text and the placeholder text, so that it's more clear what is a placeholder and what isn't.